hm_pyhelper/hardware_definitions.py

```diff
@@ -242,6 +264,8 @@ def is_raspberry_pi() -> bool:
     # Nebra Hotspot HAT RPi 3/4 Full
     'NEBHNT-HHRPI': {
         'FRIENDLY': 'Nebra Hotspot HAT RPi',
+        'CPU_ARCH': 'aarch64',
```

Comment:
This currently says raspberry pi 3/4 so maybe should be split into to as they will have different Balena device type

Reply:
Also potentially could be removed as we have never made this hardware

Reply:
removed